tests/ast/conftest.py

```diff
@@ -42,6 +43,108 @@ def pytest_configure(config):
     pytest.update_expectations = config.getoption("--update-expectations")
 
 
+class Tables:
```

Comment:
```suggestion
class TestTables:
```

Reply:
I agree that `TestTables`, or a similar name that describes their nature would be better. I'm not deeply familiar with Pytest - do you happen to know whether it will interpret the class name in any way? Normally, it picks up all `test*` and `Test*` entities and tries to use them as test classes/functions. This is why I avoided both prefixes in all names here.